.circleci/config.yml

```diff
@@ -107,6 +107,34 @@ jobs:
       - run:
           name: Type checking
           command: yarn run test:types
+  test_types_no_react:
+    <<: *defaults
+    steps:
+      - checkout
+      - run: *install_yarn_version
+      - restore_cache: *restore_yarn_cache
+      - run:
+          name: Remove React types
+          command: |
+            # This simulates @types/react not being installed
+            node -e "
+              fs.writeFileSync(
+                './package.json',
+                JSON.stringify(
+                  Object.assign(require('./package.json'), {
+                    resolutions: { '@types/react': 'file:./scripts/empty-package' },
+                  }),
+                  null,
+                  2
+                )
+              );
+            "
+            # not running the scripts, as they should not run for *-react
+            yarn install --ignore-scripts
```

Comment:
Should we also do it for Preact types, in case we install them in the monorepo in the future?

Reply:
Yes, for now that wouldn't have an impact, but I'll add the alias